src/bundles/ipfs-desktop.js

```diff
@@ -0,0 +1,8 @@
+const bundle = {
+  name: 'ipfs-desktop',
+  reducer: (state = {}) => state,
+  selectHasIpfsDesktop: () => !!window.ipfsDesktop,
```

Comment:
minor but I think the api for the rest of the app should be `selectIsIpfsDesktop`. I can see that it's named becuase it's checking for the presence of `window.ipfsDesktop`, but that is an implementation detail that may change, while the rest of the app only cares if it **is** "desktop" or not.